<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignalSelector</class>
 <widget class="QWidget" name="SignalSelector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>98</width>
    <height>204</height>
   </rect>
  </property>
  <layout class="QFormLayout" name="layout">
   <property name="rowWrapPolicy">
    <enum>QFormLayout::WrapAllRows</enum>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="cbNodes_label">
     <property name="text">
      <string>Node</string>
     </property>
     <property name="buddy">
      <cstring>cbNodes</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="cbNodes"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="cbMessages_label">
     <property name="text">
      <string>Message</string>
     </property>
     <property name="buddy">
      <cstring>cbMessages</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="cbMessages"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="cbSignals_label">
     <property name="text">
      <string>Signal</string>
     </property>
     <property name="buddy">
      <cstring>cbSignals</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="cbSignals"/>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="confirmButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Confirm</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
